publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/dynamodbstream/ImportCandidateDynamoDbStreamToEventBridgeHandlerTest.java

```diff
@@ -102,18 +104,26 @@ void shouldThrowExceptionWhenWritingEventToS3Fails() {
     }
 
     @Test
-    void shouldThrowExceptionWhenFailingOnSerializationOfEventBridgeDetails() {
-        var event =
```

Comment:
Not sure how can I trigger this error now, for we are now accepting Resource only.